project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.18
+sbt.version=1.3.4
```

Comment:
There is now a later version which we might aswell upgrade to. I believe I used 1.3.13 for VACLUF.